django/contrib/auth/models.py

```diff
@@ -167,6 +167,24 @@ def create_superuser(self, username, email, password, **extra_fields):
 
         return self._create_user(username, email, password, **extra_fields)
 
+    def with_perm(self, perm, is_active=True, is_superuser=True, backend=None, obj=None):
+        if backend is None:
+            backends = auth._get_backends(return_tuples=True)
+            if len(backends) != 1:
+                raise ValueError(
+                    'You have multiple authentication backends configured and '
+                    'therefore must provide the `backend` argument.'
```

Comment:
Do we use backticks in exception messages like this @felixxm? Not sure on the convention...

Reply:
Last time I worked on Django, the use of backticks was OK. I can change them to 'backend' if needed.

Reply:
I think it's ok to leave backticks.